Cargo.toml

```diff
@@ -64,6 +64,9 @@ log = "0.4.11"
 env_logger = "0.8"
 dirs = { version = "3.0", optional = true }
 mime_guess = { version = "2.0", optional = true }
+rand = "0.7"
+ed25519-dalek = "1.0.1"
```

Comment:
Nit: Unless there is a need for us to pin to a specific patch version here and with `base64`, I'd drop the patch number